<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planilhas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(125, 0, 39);</string>
  </property>
  <widget class="QComboBox" name="comboBox_nomes_tabelas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>614</y>
     <width>241</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Selecione uma Tabela abaixo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_importar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>630</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Importar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_carregar_tabela">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>628</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 102, 204);
</string>
   </property>
   <property name="text">
    <string>Carregar Tabela</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exportar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>630</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 102, 204);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_planilhas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1071</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
